core/src/main/res/values/strings.xml

```diff
@@ -3,6 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingTranslation">
 
+
+
```

Comment:
Can you please remove the empty lines here?

Reply:
sorry, was unavailable for a couple days.  Got those empty lines removed.

Glad it's looking good. I'll post the screenshots next time and try not to overthink it too much